packages/scss/src/components/inputs/textfield/_textfield.scss

```diff
@@ -234,8 +217,8 @@
 			color: _color("text.light");
 			@include makeIcon("search");
 			position: absolute;
-			right: 0;
-			bottom: .5rem;
+			bottom: .2rem;
```

Comment:
As stipulated before, it could be useful to use a variable to have the same positionning for suffixes and those icons.